lib/compute.rb

```diff
@@ -0,0 +1,29 @@
+module Declarations
+  module prefills
+    module Prefillables
+      class MonthlyVatReturnService < ApplicationService
+        SENTRY_OWNER = Constants::SentryOwners::DECLARATIONS
+
+        private
+
+        IMPORTATION_VAT_RATES = [0.2, 0.1, 0.085, 0.055, 0.021, 0.0105].freeze
```

Comment:
Consider extracting the `IMPORTATION_VAT_RATES` constant into a separate module or class to avoid cluttering the service class and to provide a single source of truth for VAT rates.